blocks/url-input/index.js

```diff
@@ -104,6 +104,10 @@ class UrlInput extends Component {
 
 	onKeyDown( event ) {
 		const { selectedSuggestion, posts } = this.state;
+		if ( ! this.state.showSuggestions || ! this.state.posts.length ) {
```

Comment:
Can we have a comment here to explain what this is for? I don't know what showSuggestions has to do with the key event.